proposal.rst

```diff
@@ -197,45 +233,88 @@ Project Details
 
 **Describe any existing work and concepts on which your project is based.**
 
-FILLME
+
+I am still trying to see if we can utilize the N-gram algorithm to help on this task.I made a google `doc <https://docs.google.com/document/d/1yXdnAhtNKWcODLLBy2hB2pluQ2nr3EfZJP2_F1XIpDs/edit
+/>`_ that I keep updated with results I reached so far.
```

Comment:
This link doesn't go anywhere. In any case, when you submit your final proposal it should contain the details of what you're proposing to do; you shouldn't link to a separate document that you can change after the submission deadline.